docs/getting-started/node.md

```diff
@@ -0,0 +1,215 @@
+  console.log(`⚡️[server]: Server is running at http://localhost:${port}`);
+});
+```
+
+### Step 3: Add the OpenFeature SDK
+
+Using NPM, install the OpenFeature SDK using the following command.
+
+```sh
+npm install @openfeature/js-sdk
+```
+
+Update `index.ts` to import the SDK.
+
+```diff
+import express from "express";
++ import { OpenFeature } from "@openfeature/js-sdk";
+
+const app = express();
+```
+
+Once you've imported `OpenFeature`, a new client can be created.
+
+```diff
+const port = 8080;
+
++ const client = OpenFeature.getClient();
+
+app.get("/", (_, res) => {
+```
+
+The client can now be used to get a feature flag value. In this case, we'll get a `boolean` value using the `welcome-message` [flag key][flag-key].
+
+The second argument is the fallback value, which is returned if there's abnormal behavior.
+
+```diff
+- app.get("/", (_, res) => {
++ app.get("/", async (_, res) => {
++   const showWelcomeMessage = await client.getBooleanValue("welcome-message", false);
++   if (showWelcomeMessage) {
++     res.send("Express + TypeScript + OpenFeature Server");
++   } else {
+-   res.send("Express + TypeScript Server");
++     res.send("Express + TypeScript Server");
++   }
+});
+
+```
+
+### Step 4: Run the application
+
+Let's start the app and see it in action. Run the following command to start the server.
+
+```sh
+npx ts-node index.ts
+```
+
+Open your favorite browser and navigate to http://localhost:8080. If all goes as planned, you should see "Express + TypeScript Server" in glorious monochrome.
+
+"Why I'm I seeing that value?", you may ask. Well, it's because a provider hasn't been configured yet. By default, OpenFeature will return the default value in that situation. In the next step, you'll learn how to add a provider.
+
+> NOTE: You should stop the app by using the keyboard short `ctrl + c` or `⌘ + c` before moving on to the next step.
+
+### Step 5: Add a provider to OpenFeature
+
+[Providers][provider-overview] are an important concept in OpenFeature because they're responsible for the flag evaluation itself. As we saw in the previous step, OpenFeature without a provider always returns the default value. If we want to actually perform feature flagging, we'll need to register a provider.
+
+There are [many providers available][js-providers] for popular feature flag management platforms and open source projects. You can also build your own provider if your favorite tool isn't supported yet.
+
+In this example, we'll use the `flagd provider` because it's a simple, self-contained feature flagging tool that's built and maintained by the OpenFeature team.
+
+Using NPM, install the `flagd provider` using the following command:
+
+```sh
+npm install @openfeature/flagd-provider
+```
+
+Now, update `index.ts` to import the `flagd provider`.
+
+```diff
+import { OpenFeature } from "@openfeature/js-sdk";
++ import { FlagdProvider } from "@openfeature/flagd-provider";
+
+const app: Express = express();
```

Comment:
we could remove this line and only show the import. IMO this makes the change clear